Campaign_Manager/models.py

```diff
@@ -1,29 +1,70 @@
 from django.db import models
 from Character_Builder.models import Character
+from django.contrib.auth.models import User
 
 # Constants
 MAX_LENGTH_CAMPAIGN_NAME = 255
 
+# Automatically creates CampaignIDs
+def newCampaignID():
```

Comment:
I don't think these newID functions are necessary. You can set an auto-incrementing primary key with:

models.AutoField(primary_key=True)

See: https://docs.djangoproject.com/en/1.11/topics/db/models/#automatic-primary-key-fields